script.js

```diff
@@ -1,12 +1,333 @@
+      makePageForEpisodes(allEpisodes);
+    });
+};
+
+function displayAllShows(showList) {
+  showList.forEach((show) => {
+    let showContainer = document.createElement("section");
+    showContainer.className = "show-container";
+    containerEl.appendChild(showContainer);
+    // console.log(show);
+
+    // showname
+    const showName = document.createElement("h3");
+    showName.className = "show-name";
+    showName.innerHTML = `${show.name}`;
+    showName.addEventListener("click", (e) => {
+      let selectedName = e.target.innerHTML;
+      let selected = allShows.filter((show) => show.name === selectedName);
+      let showID = selected[0].id;
+      containerEl.innerHTML = "";
+      loadShowList(showID);
+      homeEl.style.display = "block";
+    });
+    showContainer.appendChild(showName);
+
+    // populating the select show
+    let optionShow = document.createElement("option");
+    optionShow.innerHTML = `${show.name}`;
+    selectShowEl.appendChild(optionShow);
+
+    // show thumbnails
+    const showImage = document.createElement("img");
+    if (show.image) showImage.src = `${show.image.medium}`;
+    else
+      showImage.src =
+        "https://img.freepik.com/premium-vector/error-404-landing-page-with-file-flat-design_249405-162.jpg?w=2000";
+    showContainer.appendChild(showImage);
+    // show ratings combo
+    const comboEl = document.createElement("section");
+    comboEl.className = "combo";
+    showContainer.appendChild(comboEl);
+
+    //show genre
+    const showGenre = document.createElement("p");
+    showGenre.innerHTML = `<strong style = "color: green">Genre(s): </strong> ${show.genres}`;
+    comboEl.appendChild(showGenre);
+
+    //show rating
+    const showRating = document.createElement("p");
+    showRating.innerHTML = `<strong style = "color: green">Ratings:</strong> ${show.rating.average}`;
+    comboEl.appendChild(showRating);
+
+    //show status
+    const showStatus = document.createElement("p");
+    showStatus.innerHTML = `<strong style = "color: green">Status:</strong> ${show.status}`;
+    comboEl.appendChild(showStatus);
+
+    //show runtime
+    const showRuntime = document.createElement("p");
+    showRuntime.innerHTML = `<strong style = "color: green">Runtime: </strong>${show.runtime}`;
+    comboEl.appendChild(showRuntime);
+
+    //show summary
+    const description = document.createElement("section");
+    description.className = "description";
+    showContainer.appendChild(description);
+    const expandMoreContent = document.createElement("section");
+    expandMoreContent.className = "expandMoreContent";
+    expandMoreContent.setAttribute("id", "showMoreContent");
+    description.appendChild(expandMoreContent);
+    const showSummary = document.createElement("p");
+    showSummary.className = "show-summary";
+    showSummary.innerHTML = `${show.summary}`;
+    expandMoreContent.appendChild(showSummary);
+    // const expandMoreHolder = document.createElement("section");
```

Comment:
Nice try here on the commented out code to create read more and read less functionality! Although, you still can make it work using JavaScript, there's also a purely CSS and a lot easier solution to it.
Check this YouTube video out: https://www.youtube.com/watch?v=OhCzEjXvY9A&list=LL&index=4
Or if you would like to do it with JavaScript, think of updating only the `innerHTML` to truncated or full summary. You can also switch the innerHTML of the same button between read more and read less following some simple logic.